ui/src/components/schemas/SchemaDetails.tsx

```diff
@@ -155,14 +147,39 @@ export function SchemaDetails() {
       title="Schema details"
     >
       {(() => {
-        if (
-          isAsyncTaskDataAvailable(apiSchema) &&
-          isAsyncTaskDataAvailable(schema) &&
-          isAsyncTaskDataAvailable(jsonLdType) &&
-          rawJsonLdContext &&
-          rawJsonSchema
-        ) {
+        if (hasAsyncTaskFailed(apiSchema)) {
+          return (
+            <Card style={{ margin: "auto", maxWidth: CARD_WIDTH }}>
```

Comment:
If you are reverting to this structure, I would insist we find a way to abstract this custom `Card`.

Reply:
Why? it's only 4 repetitions, very localized and the code makes total sense

Reply:
Solution agreed on a meet.